user-interface/src/main/java/life/qbic/datamanager/views/general/footer/AboutUs.java

```diff
@@ -0,0 +1,90 @@
+package life.qbic.datamanager.views.general.footer;
```

Comment:
Please remove the about us page until we have content and not stalling this PR unnecessary

Reply:
done